### ISSUE TITLE
Running cargo test executes 0 test

### ISSUE THREAD

**Issue by ReinierMaas** [blt/bughunt-rust#11]
I thought to run the `cargo test` command but no test are actually executed.
latest stable: rustc 1.30.0 & cargo 1.30

**Comment by blt** [comment]
I'll document how to run the fuzz targets and ping you on the PR. I wish it were as straightforward as `cargo test`.

**Comment by ReinierMaas** [comment]
Your readme made it seem that easy!

I would love to help improve the safety guarantees of Rust.

**Comment by blt** [comment]
That's my fault, yeah. I neglected to update the README. A while back I transitioned away from QuickCheck to experiment with a model driven fuzzer, approach outlined here: https://blog.troutwine.us/2018/10/08/hunting-for-bugs-in-rust/ I never updated the README. Will correct.

**Comment by ReinierMaas** [comment]
What I would actually like to test is `time QUICKCHECK_GENERATOR_SIZE=1000 QUICKCHECK_TESTS=10000 cargo test --release`.
This way the tested code is optimized before execution which would reduce the number of branches if it can be proven by the compiler.

**Comment by ReinierMaas** [comment]
Yes that is the blog post I found. I stumbled on this project earlier but it regained my attention!

**Comment by blt** [comment]
Hello @ReinierMaas, in support of #15 I've adapted the project to the use of cargo-fuzz. I've updated the readme with running instructions and it should be easy again to run individual tests. Sadly `cargo test` won't work but `cargo fuzz run hash_map` will.
